FrontEnd/src/App/ConsensusTracker.java

```diff
@@ -0,0 +1,33 @@
+package App;
+
+import java.util.HashMap;
+
+public class ConsensusTracker {
+	private HashMap<Integer, RequestConsensus> requests = new HashMap<>();
+	private int consensusCountNeeded;
+	
+	public ConsensusTracker(int consensusCountNeeded) {
+		this.consensusCountNeeded = consensusCountNeeded;
```

Comment:
Shouldn't this always be two?

Reply:
Nope. In theory, since we have 4 replicas, we need to get 3 identical answers in order to get a majority answer. We could have a scenario of 2 against 2. That's why I initialized the consensusCountNeeded to 3.

In practice, however, we know only 1 replica will give the wrong answer. So... we could set it to 2.

I don't know what you prefer.

Reply:
you answered why this needs to be a variable im good with it